content-style-guide/01-voice-and-tone.md

```diff
@@ -0,0 +1,98 @@
+# Voice and Tone
+
+The content we make available on [Node.js](https://nodejs.org/en/) is for many different people and teams. They will use it to make decisions about teaching themselves Node.js, using it for their projects, and about contributing to its development.
+
+We can write content that empowers their decision by being aware of our voice and our tone. Now, let's get to it. What’s the difference between voice and tone?
+
+## Voice vs tone
+
+Your voice is the core of your personality. Voice is consistent across different
+contexts, and a well-crafted voice communicates both intent and values.
+
+Your tone will adapt to the emotional state of the person or audience for whom you’re
+writing. It would not be a good idea to use the same tone of voice with someone who’s
+upset as you would with someone who’s happy.
+
+Our voice doesn’t change much from day to day, but our tone changes all the time.
+
```

Comment:
A short, one-sentence comparison of tone vs voice with a little allegory or something here would be cool!

Reply:
I'm blanking on this right now 🤦🏽‍♂️

Reply:
Got one.